deploy-manage/deploy/self-managed/install-kibana-with-debian-package.md

```diff
@@ -80,15 +80,33 @@ Examine `/etc/apt/sources.list.d/kibana-9.x.list` for the duplicate entry or loc
 
 The Debian package for {{kib}} {{version.stack}} can be downloaded from the website and installed as follows:
 
+::::{tab-set}
+
+:::{tab-item} Latest
+To download and install the {{kib}} {{version.stack}} package, enter:
 ```sh subs=true
 wget https://artifacts.elastic.co/downloads/kibana/kibana-{{version.stack}}-amd64.deb
 shasum -a 512 kibana-{{version.stack}}-amd64.deb <1>
 sudo dpkg -i kibana-{{version.stack}}-amd64.deb
 ```
 
 1. 	Compare the SHA produced by shasum with the [published SHA](https://artifacts.elastic.co/downloads/kibana/kibana-9.0.0-amd64.deb.sha512).
+% version manually specified in the link above
+:::
+
+:::{tab-item} Specific version
+Replace `<SPECIFIC.VERSION.NUMBER>` with the {{kib}} version number you want to install. For example, you can replace `<SPECIFIC.VERSION.NUMBER>` with {{version.stack.base}}.
```

Comment:
I think we want the same note about this needing to be the same version here.

Reply:
Fixed, thank you